tests/Aspire.Hosting.Tests/ExpressionResolverTests.cs

```diff
@@ -130,7 +130,7 @@ public TestExpressionResolverResource(string exprName) : base("testresource")
 
         Expressions = new()
         {
-            { "TwoFullEndpoints", ReferenceExpression.Create($"Test1={Endpoint1.Property(EndpointProperty.Scheme)}://{Endpoint1.Property(EndpointProperty.IPV4Host)}:{Endpoint1.Property(EndpointProperty.Port)}/;Test2={Endpoint2.Property(EndpointProperty.Scheme)}://{Endpoint2.Property(EndpointProperty.Host)}:{Endpoint2.Property(EndpointProperty.Port)}/;") },
```

Comment:
Can you add more tests instead of changing these?

Reply:
Will do - may not be until later this week I'm afraid.

Reply:
I can merge and you can follow up 😄

Reply:
Works for me.